content/help/faq/security/mysql-with-mtls.md

```diff
@@ -0,0 +1,60 @@
+---
+title: MySQL Connectivity Troubleshooting
+description: Integrate VMs and bare metal hosts into an Istio mesh deployed on Kubernetes.
+weight: 95
+keywords: [mysql,mtls]
+---
+
+# MySQL Connectivity Troubleshooting
+
```

Comment:
No need for this header, the infrastructure supplies the title based on the content of title: above

Reply:
Removed.